i18n/de.toml

```diff
@@ -205,15 +205,15 @@ priceLimit = "Preisgrenze von {price}"
 remove = "Entfernen"
 setTargetTime = "keine"
 targetIsInThePast = "Wähle einen Zeitpunkt in der Zukunft, Marty."
-targetIsTooFarInTheFuture = "Wir passen den Plan an sobald wir mehr über die Zukunft wissen."
+targetIsTooFarInTheFuture = "Wir passen den Plan an, sobald wir mehr über die Zukunft wissen."
```

Comment:
Danke :)

Reply:
@andig Bitte, gern. Siehe: https://www.duden.de/rechtschreibung/sobald

Reply:
Komma in Weblate angepasst. Prozent in eigenem Issue.